app/views/layouts/application_template.html.erb

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en-US">
+
+  <head>
+    <meta charset="UTF-8">
+    <title><%= yield :template %> | Madeline System</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= yield :head %>
+  </head>
+
+  <body>
+    <%= render "page/page" %>
+  </body>
+
+</html>
```

Comment:
Is there a reason you made a separate file "application_template" that's incorporated into "application"? I think you could merge the two and simplify things, unless there's something I'm missing.